src/App.js

```diff
@@ -1,16 +1,18 @@
 import React, { Component } from 'react';
-import './App.scss';
+import ButtonPage from './ButtonPage.js';
+import HeaderBar from './HeaderBar.js';
 
 class App extends Component {
   render() {
+    if (window.location.pathname === '/button.html') {
```

Comment:
It might be more readable to use a hash map instead of `if` statements.

Does JSX syntax support something like this?

```jsx
const urlToComponentFactoryMap = {
  '/button.html': () => <ButtonPage />,
};

class App extends Component {
  render() {
    const componentFactory = urlToComponentFactoryMap[window.location.pathname];
    if (componentFactory) {
      return componentFactory();
    }

    return (
      <div>
        <HeaderBar title='Material Components Web | Catalog'/>
        <a href='/button.html'>Button</a>
      </div>
    );
  }
}
```